DockDoor/Views/Hover Window/SharedPreviewWindowCoordinator.swift

```diff
@@ -162,20 +162,34 @@ final class SharedPreviewWindowCoordinator: NSWindow {
         let screenFrame = screen.frame
         let dockPosition = DockUtils.getDockPosition()
 
+        // Flip the coordinate space from the accessibility API (origin is bottom-left)
+        let flippedIconRect = CGRect(
+            x: iconRect.origin.x,
+            y: screenFrame.height - iconRect.origin.y - iconRect.height,
```

Comment:
I have some helper methods called "cgRectFromNSRect" and vice versa which can do this for you.